src/main/java/frc/robot/RobotContainer.java

```diff
@@ -241,30 +234,41 @@ public void autoChooserUpdate() {
+
+      m_autoCommand.addOption("ROUTINE 19: Grid 1 Intake Cone Top", new Deposit(Position.TOP, m_elevator, m_wrist, m_intake).andThen(new PathPlannerCommand("Grid 1 Mobility", 0, m_drive, true)).andThen(new IntakeGamePiece(m_intake, () -> true, false)));
+      m_autoCommand.addOption("ROUTINE 20: Grid 1 Intake Cube Top", new Deposit(Position.TOP, m_elevator, m_wrist, m_intake).andThen(new PathPlannerCommand("Grid 1 Mobility", 0, m_drive, true)).andThen(new IntakeGamePiece(m_intake, () -> false, false)));
+
+      m_autoCommand.addOption("ROUTINE 21: Grid 9 Intake Cone Top", new Deposit(Position.TOP, m_elevator, m_wrist, m_intake).andThen(new PathPlannerCommand("Grid 9 Intake", 0, m_drive, true)).andThen(new IntakeGamePiece(m_intake, () -> true, false)));
+      m_autoCommand.addOption("ROUTINE 22: Grid 9 Intake Cube Top", new Deposit(Position.TOP, m_elevator, m_wrist, m_intake).andThen(new PathPlannerCommand("Grid 9 Intake", 0, m_drive, true)).andThen(new IntakeGamePiece(m_intake, () -> false, false)));
 
-      m_autoCommand.addOption("ROUTINE 16: Grid 9 Engage Hybrid", new DepositThenPath("Grid 9 Engage", Position.BOTTOM, doPath, m_drive, m_elevator, m_wrist, m_intake).andThen(new BalanceCommand(m_drive)));
-      m_autoCommand.addOption("ROUTINE 17: Grid 9 Engage Mid", new DepositThenPath("Grid 9 Engage", Position.MIDDLE, doPath, m_drive, m_elevator, m_wrist, m_intake).andThen(new BalanceCommand(m_drive)));
-      m_autoCommand.addOption("ROUTINE 18: Grid 9 Engage Top", new DepositThenPath("Grid 9 Engage", Position.TOP, doPath, m_drive, m_elevator, m_wrist, m_intake).andThen(new BalanceCommand(m_drive)));
+      m_autoCommand.addOption("ROUTINE 23: Grid 1 Two Piece Cone Top", new Deposit(Position.TOP, m_elevator, m_wrist, m_intake).andThen(new PathPlannerCommand("Grid 1 Mobility", 0, m_drive, true)).andThen(new IntakeGamePiece(m_intake, () -> true, false)).andThen(new PathPlannerCommand("Grid 1 Deposit", 0, m_drive, true)));
+      m_autoCommand.addOption("ROUTINE 24: Grid 1 Two Piece Cube Top", new Deposit(Position.TOP, m_elevator, m_wrist, m_intake).andThen(new PathPlannerCommand("Grid 1 Mobility", 0, m_drive, true)).andThen(new IntakeGamePiece(m_intake, () -> false, false)).andThen(new PathPlannerCommand("Grid 1 Deposit", 0, m_drive, true)));
 
+      m_autoCommand.addOption("ROUTINE 25: Grid 9 Two Piece Cone Top", new Deposit(Position.TOP, m_elevator, m_wrist, m_intake).andThen(new PathPlannerCommand("Grid 9 Intake", 0, m_drive, true)).andThen(new IntakeGamePiece(m_intake, () -> true, false)).andThen(new PathPlannerCommand("Grid 9 Deposit", 0, m_drive, true)));
+      m_autoCommand.addOption("ROUTINE 26: Grid 9 Two Piece Cube Top", new Deposit(Position.TOP, m_elevator, m_wrist, m_intake).andThen(new PathPlannerCommand("Grid 9 Intake", 0, m_drive, true)).andThen(new IntakeGamePiece(m_intake, () -> false, false)).andThen(new PathPlannerCommand("Grid 9 Deposit", 0, m_drive, true)));
```

Comment:
In addition the the intake stuff I mentioned, these paths also don't outtake. I also think instead of making a seperate path you can split a path in two by adding a stop point. then you can run the same path but use 1 as the index to access the next path.
Do you want to add them as a separate PR?

Reply:
just made the changes in this pr